mail-client/alpine/alpine-2.21.1-r2.ebuild

```diff
@@ -0,0 +1,94 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit autotools toolchain-funcs
+
+DESCRIPTION="An easy to use text-based based mail and news client"
+HOMEPAGE="http://www.washington.edu/alpine/ http://repo.or.cz/alpine.git/"
+GIT_COMMIT="843b2f16abfd949e09b1c5465387b1b0f724994a"
+MY_P="${PN}-${GIT_COMMIT::7}"
+SRC_URI="http://repo.or.cz/alpine.git/snapshot/${GIT_COMMIT}.tar.gz -> ${MY_P}.tar.gz"
+S="${WORKDIR}/${MY_P}"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~ia64 ~ppc ~sparc ~x86"
+IUSE="doc ipv6 kerberos ldap libressl nls onlyalpine pam passfile smime spell ssl threads"
+
+DEPEND="pam? ( virtual/pam )
+	>=sys-libs/ncurses-5.1:0=
+	ssl? (
+		!libressl? ( dev-libs/openssl:0= )
+		libressl? ( dev-libs/libressl:0= )
+	)
+	ldap? ( net-nds/openldap )
+	kerberos? ( app-crypt/mit-krb5 )
+	spell? ( app-text/aspell )
+"
+RDEPEND="${DEPEND}
+	app-misc/mime-types
+	!onlyalpine? ( !mail-client/pine )
+	!<=net-mail/uw-imap-2004g
+"
+
+src_prepare() {
+	sed -ie 's/AC_CHECK_LIB[^(]*([^,]*pam[^,]*,[^,]*,/AS_IF(['$(
+		usex pam true false)'],/' -- "${S}"/configure.ac
```

Comment:
missing `|| die`. However, a change like that should be done as a patch instead, in a form that you can upstream (providing a configure switch that you make use of below in `src_configure()`).